src/util/config.tsx

```diff
@@ -13,7 +13,9 @@ export const toConfigFields = (
           category: categoryKey,
           default: value.defaultdesc?.startsWith("`")
             ? value.defaultdesc.split("`")[1]
-            : "",
+            : value.defaultdesc?.startsWith('"')
```

Comment:
I'm lacking a bit of context here. Can we maybe add a comment here to indicate the possible format for `value.defaultdesc`?

Reply:
This was necessary because of a wrong format in LXD docs. I fixed it with this pr https://github.com/canonical/lxd/pull/14502#pullrequestreview-2450170732

So will remove this workaround before merge

Reply:
Oh nice! :+1: